apps/admin/frontend/test/helpers/api_expect_helpers.ts

```diff
@@ -0,0 +1,57 @@
+import { Tabulation } from '@votingworks/types';
+import { getEmptyCardCounts } from '@votingworks/utils';
+import { ApiMock } from './api_mock';
+
+/**
+ * To match backend input exactly, per `MockFunction` requirements, mock out
+ * false values instead of undefined.
+ */
+export function mockBallotCountsTableGroupBy(
+  groupBy: Tabulation.GroupBy
+): Tabulation.GroupBy {
+  return {
+    groupByBatch: false,
+    groupByParty: false,
+    groupByPrecinct: false,
+    groupByScanner: false,
+    groupByVotingMethod: false,
+    ...groupBy,
+  };
+}
```

Comment:
This is necessary because `grout`'s `deepEqual` naturally does not consider `false` and `undefined` as the same.